cmd/juju/crossmodel/remove.go

```diff
@@ -96,12 +121,25 @@ func (c *removeCommand) Run(ctx *cmd.Context) error {
 		}
 		c.offerSource = controllerName
 	}
+
+	if !c.assumeYes && c.force {
+		fmt.Fprintf(ctx.Stdout, removeOfferMsg, strings.Join(c.offers, ", "))
+
+		if err := jujucmd.UserConfirmYes(ctx); err != nil {
+			return errors.Annotate(err, "offer removal")
+		}
+	}
+
 	api, err := c.newAPIFunc(c.offerSource)
 	if err != nil {
 		return errors.Trace(err)
 	}
 	defer api.Close()
 
-	err = api.DestroyOffers(c.offers...)
+	if api.BestAPIVersion() < 2 {
```

Comment:
Are you sure that you want to do this? 
I feel like it is more natural to support a call on api version v1, just without a 'force'...

Reply:
Ah FFS I had meant to use the force flag in the check here but forgot. Thank you.